src/fsharp/Interactive.DependencyManager/NativeDllResolveHandler.fs

```diff
@@ -14,6 +14,28 @@ open Internal.Utilities.FSharpEnvironment
 type NativeResolutionProbe = delegate of Unit -> IEnumerable<string>
 
 
+module internal RidHelpers =
+
+    // Computer valid dotnet-rids for this environment:
+    //      https://docs.microsoft.com/en-us/dotnet/core/rid-catalog
+    //
+    // Where rid is: win, win-x64, win-x86, osx-x64, linux-x64 etc ...
+    let probingRids, baseRid, platformRid =
```

Comment:
Any way to combine this with the duplicate implementation above?

Reply:
No ... this is the DependencyManagerDll, whereas, the other location is FSharp.Compiler.Private.